test/smoke/src/sql/areas/notebook/notebook.test.ts

```diff
@@ -132,6 +133,51 @@ export function setup(opts: minimist.ParsedArgs) {
 			await app.code.dispatchKeybinding('escape');
 		});
 
+		describe('Notebook keyboard navigation', async () => {
+			it('can enter and exit edit mode and navigate using keyboard nav', async function () {
+				const app = this.app as Application;
+				await app.workbench.sqlNotebook.newUntitledNotebook();
+				await app.workbench.sqlNotebook.addCellFromPlaceholder('Code'); // add new code cell
+				await app.workbench.sqlNotebook.waitForPlaceholderGone();
+				const activeCodeCellId = (await app.workbench.sqlNotebook.getActiveCell()).attributes['id'];
+				await app.workbench.sqlNotebook.waitForTypeInEditor('code cell', activeCodeCellId); // the new cell should be in edit mode
+				await app.workbench.sqlNotebook.exitActiveCell();  // hitting escape twice deselects all cells
```

Comment:
You should avoid comments that refer to implementation details. In this case if we change how we exit active cells in the future we'll have to remember to go update this comment.

And in generally we don't really care in the test how we exit the active cell - only that we're trying to do so. So I don't think a comment here is necessary at all. The function name is clear enough.